common/model/transaction.h

```diff
@@ -13,6 +13,8 @@ extern "C" {
 #define __COMMON_MODEL_TRANSACTION_H_
 
 #include <assert.h>
+#include <inttypes.h>
+#include <stdio.h>
```

Comment:
not needed

Reply:
for debugging.

Reply:
you can include these in the .c though? no need to depend on them for all includers of this header